src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,20 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Machine> {
+    public ExcavatorProducer() {
+    }
+
+    @Override
+    public List<Machine> get() {
```

Comment:
Return a list of `Excavator` instead of `Machine`. Since this producer is specific to `Excavator`, the list should be explicitly of that type.